### ISSUE TITLE
Copy searched text problem

### ISSUE THREAD

**Issue by jhhuisman** [qutebrowser/qutebrowser#5272]
**Version info**:
qutebrowser v1.10.1
Git commit: 
Backend: QtWebEngine (Chromium 56.0.2924.122)
Qt: 5.9.5

CPython: 3.6.9
PyQt: 5.10.1

sip: 4.19.7
colorama: no
pypeg2: 2.15
jinja2: 2.10
pygments: 2.2.0
yaml: 3.12
cssutils: no
attr: 17.4.0
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebEngine: yes
PyQt5.QtWebKitWidgets: no
pdf.js: no
sqlite: 3.22.0
QtNetwork SSL: OpenSSL 1.1.1d  10 Sep 2019

Style: QFusionStyle
Platform: Linux-4.15.0-88-generic-x86_64-with-Ubuntu-18.04-bionic, 64bit
Linux distribution: Ubuntu 18.04.4 LTS (ubuntu)
Frozen: False
Imported from /usr/lib/python3/dist-packages/qutebrowser
Using Python from /usr/bin/python3
Qt library executable path: /usr/lib/x86_64-linux-gnu/qt5/libexec, data path: /usr/share/qt5

**Does the bug happen if you start with `--temp-basedir`?**:
Yes

**Description**
I would like to copy (yank) text from a page.
I try to search for part of the string using / and then press enter.
Then I press v to enter caret mode.
I expect to enter 'Caret selection mode' and be able to yank the text using y.
However what happen is I enter regular 'Caret mode' and the search result disappears.
Can someone please help me with this?

It is working as expected if I select text using the mouse and then press v.
I also notice that search results are colored yellow and orange (for current match).
Mouse selections and 'Caret selection mode' selections are colored blue.

(Posted on Reddit yesterday: https://www.reddit.com/r/qutebrowser/comments/fe2qbf/copy_searched_text_problem/)

Gif of what happens:
https://i.imgur.com/GoKf7rG.gif

**Comment by The-Compiler** [comment]
Whether a search is converted to a selection or not depends on the Qt version you're using. In versions before Qt 5.10, this didn't work reliably, see [[QTBUG-60673] Race Condition when Clearing Search to generate a selection - Qt Bug Tracker](https://bugreports.qt.io/browse/QTBUG-60673) - nothing qutebrowser can do about that, though.

**Comment by jhhuisman** [comment]
That's a quick reply. Thank you so much.
I was suspecting this would be the issue as I noticed higher Qt and QtWebEngine versions in other issues here.
Can you help me update those? I am using Ubuntu 18.04. I hope that's not too much to ask. Thanks in advance.

**Comment by The-Compiler** [comment]
If you don't need proprietary video codecs, you might want to look at https://github.com/qutebrowser/qutebrowser/blob/master/doc/install.asciidoc#tox - if you do, you'd need to build Qt/QtWebEngine and PyQt from source which can be a bit of a pain.

**Comment by jhhuisman** [comment]
Awesome that was not too hard.
It fixed my problem from this ticket.
Thank you so much.

**Comment by jhhuisman** [comment]
I have another question if you don't mind.

Today I wanted to run Qutebrowser again using:
~/qutebrowser/.venv/bin/python3 -m qutebrowser "$@"

But it did not run. I got this error:
/home/jacob/qutebrowser/.venv/bin/python3: No module named qutebrowser.__main__; 'qutebrowser' is a package and cannot be directly executed

I had to rerun:
python3 scripts/mkvenv.py

Then wait for a bit to install again.
Do I have to rerun this everytime I reboot? I rather not.
Thanks in advance.
